src/components/SelectCircle.js

```diff
@@ -9,15 +9,20 @@ import themeColors from '../styles/themes/default';
 const propTypes = {
     /** Should we show the checkmark inside the circle */
     isChecked: PropTypes.bool,
+
+    /** Additional styles to pass to SelectCircle */
+    // eslint-disable-next-line react/forbid-prop-types
+    styles: PropTypes.arrayOf(PropTypes.object),
 };
 
 const defaultProps = {
     isChecked: false,
+    styles: [],
 };
 
 function SelectCircle(props) {
     return (
-        <View style={[styles.selectCircle, styles.alignSelfCenter]}>
+        <View style={[styles.selectCircle, styles.alignSelfCenter, props.styles]}>
```

Comment:
Shouldn't props.styles be spread since it's an array?

Reply:
Done